taplo.toml

```diff
@@ -0,0 +1,6 @@
+[formatting]
+align_comments = true
```

Comment:
question: This is already the default. Is there a reason you want to specify it explicitly?

Reply:
Dunno :) maybe I liked seeing both align options